go/Dockerfile.build

```diff
@@ -0,0 +1,10 @@
+FROM golang:1.8-alpine
+RUN apk update && apk add --no-cache make git
+ENV GOPATH=/go
+ENV PATH=/go/bin:/usr/local/go/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+
+# The project sources
+VOLUME ["/go/src/github.com/docker/hyperkit/go"]
+WORKDIR /go/src/github.com/docker/hyperkit/go
+
+ENTRYPOINT ["make"]
```

Comment:
Is a Linux build of this functionality useful given it can't fork+exec hyperkit in its present form?